src/components/Calculator.js

```diff
@@ -1,47 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.handleClick = this.handleClick.bind(this);
-  }
-
-  handleClick(event) {
-    this.setState((state) => calculate(state, event.target.textContent));
-  }
-
-  render() {
-    const { next, total } = this.state;
-    return (
-      <>
-        <div className="calculator-container">
-          <div className="calc-input"><span className="result">{next || total || 0}</span></div>
-          <div className="buttons">
-            <button type="button" className="bg-1" onClick={this.handleClick}>AC</button>
-            <button type="button" className="bg-1" onClick={this.handleClick}>+/-</button>
-            <button type="button" className="bg-1" onClick={this.handleClick}>%</button>
-            <button type="button" className="bg-1 active" onClick={this.handleClick}><span>&#247;</span></button>
-            <button type="button" className="bg-1" onClick={this.handleClick}>7</button>
-            <button type="button" className="bg-1" onClick={this.handleClick}>8</button>
-            <button type="button" className="bg-1" onClick={this.handleClick}>9</button>
-            <button type="button" className="bg-1 active" onClick={this.handleClick}>x</button>
-            <button type="button" className="bg-1" onClick={this.handleClick}>4</button>
-            <button type="button" className="bg-1" onClick={this.handleClick}>5</button>
-            <button type="button" className="bg-1" onClick={this.handleClick}>6</button>
-            <button type="button" className="bg-1 active" onClick={this.handleClick}>-</button>
-            <button type="button" className="bg-1" onClick={this.handleClick}>1</button>
-            <button type="button" className="bg-1" onClick={this.handleClick}>2</button>
-            <button type="button" className="bg-1" onClick={this.handleClick}>3</button>
-            <button type="button" className="bg-1 active" onClick={this.handleClick}>+</button>
-            <button type="button" className="zero" onClick={this.handleClick}>0</button>
-            <button type="button" className="bg-1" onClick={this.handleClick}>.</button>
-            <button type="button" className="bg-1 active" onClick={this.handleClick}>=</button>
-          </div>
+function Calculator() {
```

Comment:
**[REQUIRED]** For the good practice of JavaScript let convert this function to `arrow function` syntax.

**[OPTIONAL]** I highly suggest you implement displaying the operator when the user clicks one of them as of the numbers displaying in the browser.